### DISCUSSION TITLE
OpenAPI(Swagger) Integration

### DISCUSSION THREAD

**Discussion by Null45** [outline/outline#2993] [Ideas]
It would be nice feature, like Confluence or Space provides.
https://marketplace.atlassian.com/apps/1219386/open-api-swagger-integration?tab=overview&hosting=cloud

**Comment by yiukamsum** [comment]
I have tried to use a http json to show the swagger doc by using plugin `swagger-ui-react`. Is that what you want? In this stage, we cannot edit the json.
![螢幕擷取畫面 2022-05-03 003630](https://user-images.githubusercontent.com/13889428/166284391-b8cbc681-c094-4088-8fe3-52c163ecb662.png)
but there still have some bugs. Maybe I will PR and we fix it together

**Comment by almereyda** [comment]
For search engine visitors, the API documentation including OpenAPI schema download is available from https://www.getoutline.com/developers.

There is also a freely licensed repository at [outline/openapi](https://github.com/outline/openapi) with resources and tooling.
